lib/omniship/newgistics.rb

```diff
@@ -0,0 +1,29 @@
+require 'omniship/newgistics/track'
+
+module Omniship
+  module Newgistics
+    LABEL = "Newgistics"
+    TRACKING_REGEX = /TODO/
```

Comment:
We might be able to get away with figuring out from 
![image](https://cloud.githubusercontent.com/assets/2627962/16013874/bb38a6e8-3154-11e6-8716-2db922707fb7.png)

```
/7(21|25)(00|01|10|20)\d{5}01\d{4}\d{2}\d{7, 29}\d/i
```

dunno if that's valid regex tho